templates/security/esm.html

```diff
@@ -8,318 +8,555 @@
 
 {% block content %}
 
-<section class="p-strip--suru-topped is-bordered">
+<section class="p-strip--image is-deep u-no-margin-bottom" style="background-image:url('https://assets.ubuntu.com/v1/18717572-05_suru2_light_glow_8K_edited.png')">
```

Comment:
If we are going to be using this strip elsewhere, it might be worth making it into it's own SCSS style

Reply:
So @lyubomir-popov asked on purpose to do this version instead of the mobile version, because the separation of tabular data makes it difficult to identify headers

Reply:
OK, but we have to do something about it to make it look less broken/ more readable